rust/src/storage/http.rs

```diff
@@ -28,15 +29,15 @@ pub fn send_request(
     request: reqwest::Request,
 ) -> Fallible<reqwest::Response> {
     debug!("> {} {}", request.method(), request.url());
-    for header in request.headers().iter() {
-        debug!("> {}: {}", header.name(), header.value_string());
+    for (name, value) in request.headers().iter() {
+        debug!("< {}: {:?}", name, value);
```

Comment:
What's with the swapped angle bracket?

Reply:
I didn't notice that this code block had that difference, so I copied the one below to save time. :joy: 

That needs to be fixed, of course.